### ISSUE TITLE
[NADIA Review] Deep Learning for Automatic Segmentation of Vestibular Schwannoma A Retrospective Study from Multi-Centre Routine MRI -- Deep learning models

### ISSUE THREAD

**Issue by allaway** [nf-osi/nadia#81]
## NADIA Study Review

🆕 New project created

| Field | Value |
|-------|-------|
| **Synapse Project** | [syn74287641](https://www.synapse.org/Synapse:syn74287641) |
| **Study Name** | Deep Learning for Automatic Segmentation of Vestibular Schwannoma A Retrospective Study from Multi-Centre Routine MRI -- Deep learning models |
| **External Accessions** | zenodo.record:10363647 |
| **Study Leads** | Unknown |
| **Assay Types** | image |
| **Disease Focus** | Neurofibromatosis type 2 |
| **Manifestation** | Vestibular Schwannoma |
| **File Count** | 4 |
| **PMID** | — |
| **DOI** | 10.5281/zenodo.10363647 |

---

## Review Checklist

Please check each item before approving:

- [ ] Study name is accurate and meaningful
- [ ] Correct disease focus and manifestation values
- [ ] Study leads (first + last/corresponding author) are correct
- [ ] Assay type(s) are correct
- [ ] File annotations look reasonable (spot-check a few files)
- [ ] Wiki summary is informative
- [ ] No sensitive or controlled-access data exposed unintentionally

---

## Actions

**To request annotation fixes**, comment with:
```
/nadia fix: <describe what needs to be changed>
```
Examples:
- `/nadia fix: disease focus should be NF2 not NF1`
- `/nadia fix: manifestation is missing — this is a plexiform neurofibroma study`
- `/nadia fix: study lead should be Jane Doe, not John Smith`

**To recheck the project status**, comment with:
```
/nadia status
```

**To approve and trigger portal provisioning**, apply the `approved` label to this issue.
This will automatically:
1. Set `resourceStatus = approved` on the Synapse project and all files
2. Add the project to the portal file view
3. Update the NADIA state table
4. Post a summary comment here

> ⚠️  Only apply `approved` when the study is ready for the public portal.
> Once provisioned, changes require manual curator action in Synapse.

---

<details>
<summary>🔧 NADIA Metadata (used by automated provisioning — do not edit)</summary>

```json
NADIA_METADATA_JSON
{
  "synapse_project_id": "syn74287641",
  "accessions": [
    "zenodo.record:10363647"
  ],
  "study_name": "Deep Learning for Automatic Segmentation of Vestibular Schwannoma A Retrospective Study from Multi-Centre Routine MRI -- Deep learning models",
  "study_leads": [
    "Unknown"
  ],
  "assay_types": [
    "image"
  ],
  "file_count": 4,
  "outcome": "new",
  "disease_focus": [
    "Neurofibromatosis type 2"
  ],
  "manifestation": [
    "Vestibular Schwannoma"
  ],
  "pmid": "",
  "doi": "10.5281/zenodo.10363647"
}
NADIA_METADATA_JSON
```

</details>

**Comment by allaway** [comment]
## NADIA Polish Curation Summary

**Polish run date:** 2026-04-16
**Schema template:** `org.synapse.nf-sourcecodetemplate-10.6.2` (selected because the files are zip archives of pre-trained nnU-Net ML models — not biological assay data but computational tools)
**Source metadata fetched from:** Zenodo record 10363647, PubMed PMID 38784680

---

### Issues identified and fixed

| Issue | Resolution |
|-------|-----------|
| `studyLeads = 'Unknown'` | Updated to `['Aaron Kujawa', 'Jonathan Shapey']` from PubMed AuthorList (first + last author) |
| `pmid` missing | Set to `38784680` (Frontiers in Computational Neuroscience, 2024) |
| `doi` was Zenodo DOI | Updated to publication DOI `10.3389/fncom.2024.1365727` |
| `institutions` missing | Set to `School of Biomedical Engineering and Imaging Sciences, King's College London` |
| `fundingAgency = 'Not Applicable (External Study)'` | Updated to `Wellcome Trust` (from PubMed GrantList) |
| `individualID` and `specimenID` set to filename prefix fragments | Removed — these fields do not apply to ML model zip archives |
| Schema binding was `bulksequencingassaytemplate` | Rebound to `sourcecodetemplate` (correct for ML model artifacts) |
| Empty `Analysis/` folder | Deleted |
| Dataset name was `Zenodo_10363647` (generic) | Updated to `Zenodo_10363647 - Deep learning segmentation models for Vestibular Schwannoma (nnU-Net)` |
| Dataset `columnIds` contained `individualID` and `specimenID` | Rebuilt from actual file annotation keys |
| File annotations missing `title`, `citation`, `license` | Added from SourceCodeTemplate schema and Zenodo/PubMed metadata |
| Wiki used "Ingested automatically" / "Auto-discovery notice" | Updated to standard NADIA wiki format with proper attribution |
| `resourceStatus` missing from Dataset entity | Added `pendingReview` |
| `studyId` missing from Dataset entity | Added `syn74287641` |

---

### Annotations set (project level)

| Field | Value | Source |
|-------|-------|--------|
| `studyLeads` | Aaron Kujawa, Jonathan Shapey | PubMed AuthorList (PMID 38784680) |
| `pmid` | 38784680 | PubMed search (Kujawa + Dorent + vestibular schwannoma) |
| `doi` | 10.3389/fncom.2024.1365727 | PubMed record |
| `institutions` | School of Biomedical Engineering and Imaging Sciences, King's College London | PubMed AuthorList affiliation |
| `fundingAgency` | Wellcome Trust | PubMed GrantList |
| `diseaseFocus` | Neurofibromatosis type 2 | Preserved (correct — vestibular schwannoma is NF2-associated) |
| `manifestation` | Vestibular Schwannoma | Preserved (correct) |
| `dataType` | image | Preserved (MRI-based study) |

### Annotations set (file level)

| Field | Value | Notes |
|-------|-------|-------|
| `fileFormat` | zip | Correct for the archive files |
| `resourceType` | tool (3 model files), experimentalData (example images) | SourceCodeTemplate enum |
| `title` | per-file descriptive title | From filename |
| `citation` | full citation string | Constructed from PubMed record |
| `license` | CC-BY 4.0 | Zenodo record `license.id = cc-by-4.0` |
| `dataSubtype` | processed | Preserved |

---

### Gap-fill result

**Tier 1 (structured repo metadata):** citation, license from Zenodo; title from filename
**Tier 2 (publication metadata):** studyLeads, pmid, doi, institutions, fundingAgency from PubMed
**Tier 3 (text reasoning):** N/A for this study (no per-sample biological fields apply)
**Tier 4 (file inspection):** N/A for zip archives

**Fields not set (genuinely not applicable):**
- `author` (SourceCodeTemplate free-text array) — left blank; authors captured in `citation` field
- `documentation` — no separate documentation URL
- `runtimePlatform` — nnU-Net runs on any Linux/CUDA system
- `programmingLanguage` — these are model weights, not source code

---

### Schema template selection rationale

This Zenodo record contains zip archives of pre-trained nnU-Net deep learning model weights and example MRI input images. The files are computational artifacts (model weights), not biological assay data. `SourceCodeTemplate` is the closest match — it supports `zip` in its `fileFormat` enum and has `resourceType = 'tool'`. Three files are tagged `tool`; the example images zip is tagged `experimentalData`.

`MRIAssayTemplate` was considered but rejected: its `fileFormat` enum does not include `zip`, and the files are model weights, not MRI scan files.

---

### Items for human review

- [ ] **NF2 disease focus scoping**: The paper studies 160 patients with "single sporadic VS" from a multi-centre clinical dataset. Most sporadic VS cases are not NF2-related. However, VS is the defining manifestation of NF2 and this tool is clearly relevant to NF2 clinical management. A data manager may wish to add a wiki note clarifying this is a pan-VS dataset (NF2 and sporadic), or consider whether `diseaseFocus` should remain `Neurofibromatosis type 2` or be supplemented/replaced.
- [ ] **No version snapshot minted**: The Synapse `POST /entity/{id}/version` endpoint returns 405 for Dataset entities (requires async table transaction). A data manager should manually mint a snapshot from the Synapse UI if needed.
- [ ] **`author` field not populated**: SourceCodeTemplate has a free-text `author` field. Full authorship is captured in the `citation` annotation; a data manager may wish to add structured author names separately.
